colossalai/inference/modeling/models/nopadding_baichuan.py

```diff
@@ -55,24 +55,19 @@ class NopadBaichuanAttention(ParallelModule):
     def __init__(
         self,
         config,
-        attn_qproj_w: torch.Tensor = None,
-        attn_kproj_w: torch.Tensor = None,
-        attn_vproj_w: torch.Tensor = None,
+        W_pack: ParallelModule = None,
```

Comment:
Yeah, it does seem better. But the state_dict of origin weight use `W_pack`. To avoid the situation of rewrite load_from_state_dict as much as possible, we choose to use `W_pack`, aligning with original implementation :)